src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -927,6 +1445,107 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		037C3CAD2991A44B00B7EEE2 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CHIP_BUILD_TOOLS = true;
+				CHIP_ROOT = "$(PROJECT_DIR)/../../..";
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
+				CODE_SIGN_IDENTITY = "-";
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = "";
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					CHIP_HAVE_CONFIG_H,
+					"CHIP_ADDRESS_RESOLVE_IMPL_INCLUDE_HEADER=<lib/address_resolve/AddressResolve_DefaultImpl.h>",
+					CONFIG_BUILD_FOR_HOST_UNIT_TEST,
+					CONFIG_USE_LOCAL_STORAGE,
+				);
+				"HEADER_SEARCH_PATHS[arch=*]" = (
+					"$(CHIP_ROOT)/examples/darwin-framework-tool",
+					"$(CHIP_ROOT)/src",
+					"$(CHIP_ROOT)/third_party/nlassert/repo/include",
+					"$(CHIP_ROOT)/zzz_generated/app-common",
+					"$(CHIP_ROOT)/third_party/nlio/repo/include",
+					"$(CHIP_ROOT)/third_party/jsoncpp/repo/include",
+					"$(CHIP_ROOT)/zzz_generated/darwin-framework-tool",
+					"$(CHIP_ROOT)/config/ios",
+					"$(CHIP_ROOT)/third_party/editline/repo/include",
+					"$(CHIP_ROOT)/src/include",
+					"$(CONFIGURATION_TEMP_DIR)/Matter.build/out/gen/include",
+					"$(CHIP_ROOT)/zzz_generated/darwin/controller-clusters",
+					"$(CHIP_ROOT)/third_party/inipp/repo/inipp",
+					"$(CHIP_ROOT)/third_party/editline/include",
+					"$(CHIP_ROOT)/examples/chip-tool",
+					"$(CHIP_ROOT)/examples/chip-tool/commands/clusters",
+					"$(CHIP_ROOT)/zzz_generated/chip-tool",
+				);
+				"LIBRARY_SEARCH_PATHS[arch=*]" = (
+					"$(CONFIGURATION_TEMP_DIR)/Matter.build/out/lib",
+					"$(CONFIGURATION_TEMP_DIR)/Matter.build/out/obj/src/app/lib",
+				);
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
+				SDKROOT = macosx;
+				STRIP_INSTALLED_PRODUCT = NO;
+				TARGETED_DEVICE_FAMILY = 4;
+				USER_HEADER_SEARCH_PATHS = "";
+				WATCHOS_DEPLOYMENT_TARGET = 9.3;
```

Comment:
Not sure we should be setting TARGETED_DEVICE_FAMILY and WATCHOS_DEPLOYMENT_TARGET here and below?